letsmeet/communities/models.py

```diff
@@ -47,6 +47,7 @@ class Community(TimeStampedModel):
     is_deleted = models.BooleanField(default=False)
 
     objects = CommunityManager()
+
     default = models.Manager()   # the default manager
```

Comment:
This is due to a bug in pycodestyle.  Should not be changed in this PR.
See issue 617 of https://github.com/PyCQA/pycodestyle